backends/graphics/windowed.h

```diff
@@ -211,6 +211,26 @@ class WindowedGraphicsManager : virtual public GraphicsManager {
 		handleResizeImpl(width, height);
 	}
 
+	/**
+	 * Adjusts the drawRect for a system "safe area".
+	 */
+	void adjustDrawRectForSafeArea(Common::Rect &drawRect) {
+		int safeAreaInsetLeft, safeAreaInsetRight, safeAreaInsetTop, safeAreaInsetBottom;
+		int scaleFactor = g_system->getHiDPIScreenFactor();
+		g_system->getSafeAreaInsets(safeAreaInsetLeft, safeAreaInsetRight, safeAreaInsetTop, safeAreaInsetBottom);
```

Comment:
Couldn't we overload a getSafeAreaInsets from WindowedGraphicsManager inseatd of adding a new function to OSystem?

Reply:
Good idea. I'll change

Reply:
@lephilousophe one has to add functions to modular-backend as well to be able to utilise the function in ThemeParser and ThemeEval then?

Reply:
hmmm, I'm not seeing how ThemeEval would get the insets...

Reply:
Ah! I missed this part. :D
So forget about what I said